modules/monitoring-creating-a-monitoringstack-object-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-a-monitoringstack-object-for-cluster-observability-operator_{context}"]
+= Creating a MonitoringStack object for Cluster Observability Operator
+
+To be able to discover the sample `ServiceMonitor` resource you created, you must create and configure a `MonitoringStack` object.
```

Comment:
```suggestion
To be able to discover the sample `ServiceMonitor` object you created using the documentation on "Specifying how a service is monitored for Cluster Observability Operator", you must create and configure a `MonitoringStack` object.
```

I'd maybe add some explanation of "the sample `ServiceMonitor` resource you created" since it's not immediately clear out of content what that would be. Can users only use this procedure with that `ServiceMonitor` object, or can it be any `ServiceMonitor` object?

Also, I'd use "object" here as well since you use  `MonitoringStack` object, or else use "resource" throughout.

Reply:
Changed per your suggestions.